tests/multiphase_flow/rotating_barge_cib.cpp

```diff
@@ -127,12 +134,7 @@ main(int argc, char* argv[])
             }
         }
     }
-
-    // Initialize PETSc, MPI, and SAMRAI.
-    PetscInitialize(&argc, &argv, NULL, NULL);
-    SAMRAI_MPI::setCommunicator(PETSC_COMM_WORLD);
-    SAMRAI_MPI::setCallAbortInSerialInsteadOfExit();
-    SAMRAIManager::startup();
+    SAMRAI_MPI::barrier();
```

Comment:
I don't think it is doing any harm, but I also don't think you need that call to `SAMRAI::barrier()`.
```suggestion
```